USAGE.md

```diff
@@ -246,3 +246,80 @@ $ aws lambda invoke --function-name import-status \
 
 {"status_code": 200, "body": {"validation":{ "status": "SUCCEEDED"}, "metadata_upload":{"status": "Pending", "errors":[]}, "asset_upload":{"status": "Pending", "errors":[]}}}
 ```
+
+## Receive Import Status updates by subscribing to our AWS SNS Topic
+
+The ARN of our SNS Topic is `arn:aws:sns:ap-southeast-2:715898075157:geostore-import-status` which
+you may choose to subscribe too.
+
+You may also choose to apply a subscription filter policy, which will filter notifications for a
+specific dataset or specific statuses.
+
+```json
+{
+  "dataset_id": ["id_1234567890", "id_0987654321"],
+  "status": ["RUNNING", "SUCCEEDED", "FAILED", "TIMED_OUT", "ABORTED"]
+}
+```
+
+Below is an example payload received when subscribed to our SNS Topic. The Message field will
+contain a json string with specific details regarding the Step Function Execution.
+
+```json
+{
+  "Type": "Notification",
+  "MessageId": "xxxx-xxxx-xxxx",
+  "TopicArn": "arn:aws:sns:ap-southeast-2:715898075157:geostore-import-status",
+  "Message": "MESSAGE BELOW",
+  "Timestamp": "2021-07-07T01:49:33.471Z",
+  "SignatureVersion": "1",
+  "Signature": "xxxxx",
+  "SigningCertURL": "https://example.com",
+  "UnsubscribeURL": "https://example.com",
+  "MessageAttributes": {
+    "dataset_id": {
+      "Type": "String",
+      "Value": "test_123456789"
+    },
+    "status": {
+      "Type": "String",
+      "Value": "SUCCEEDED"
+    }
+  }
+}
+```
+
+Below is an example of the EventBridge 'Step Functions Execution Status Change' payload. This will
+be formatted as a string and associated with the 'Message' attribute in the above SNS payload. See
+[AWS EventBridge Payload Examples](https://docs.aws.amazon.com/step-functions/latest/dg/cw-events.html#cw-events-events)
+
+```json
+{
+  "version": "0",
+  "id": "3afc69d1-1291-6372-ae76-9a9c11ee35c4",
+  "detail-type": "Step Functions Execution Status Change",
+  "source": "aws.states",
+  "account": "715898075157",
+  "time": "2021-07-07T01:49:29Z",
+  "region": "ap-southeast-2",
+  "resources": ["arn:aws:states:example"],
+  "detail": {
+    "executionArn": "arn:aws:states:example",
```

Comment:
Is this ARN actually in the "states" namespace? If not it might be better to ellipsise it (`[…]`) or use the real namespace.

Reply:
yes it is.